src/applications/caregivers/tests/unit/components/FormFields/FacilitySearch.unit.spec.js

```diff
@@ -405,19 +408,83 @@ describe('CG <FacilitySearch>', () => {
           expect(selectors().loader).to.not.exist;
           expect(selectors().input).to.not.have.attr('error');
           expect(getByText(/Showing 1-1 of 1 facilities for/)).to.exist;
+          expect(selectors().ariaLiveStatus.textContent).to.eq('');
         });
 
         await waitFor(() => {
           userEvent.click(selectors().moreFacilities);
           expect(selectors().radioList).to.exist;
           expect(selectors().loader).to.exist;
+          expect(selectors().ariaLiveStatus.textContent).to.eq('');
         });
 
         await waitFor(() => {
           expect(selectors().radioList).to.exist;
           expect(selectors().loader).to.not.exist;
           expect(getByText(/Showing 1-2 of 2 facilities for/)).to.exist;
           expect(selectors().input).to.not.have.attr('error');
+          expect(selectors().ariaLiveStatus.textContent).to.eq(
+            '1 new facility loaded',
+          );
+        });
+
+        await waitFor(() => {
+          sinon.assert.calledWith(mapboxStub, 'Tampa');
+          sinon.assert.calledWith(facilitiesStub, {
+            lat,
+            long,
+            perPage,
+            radius,
+            page: 2,
+          });
+          expect(mapboxStub.callCount).to.equal(1);
+          // This is 3 because there is an existing bug that using submit with the va-search-input component triggers two requests
+          // https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3117
+          expect(facilitiesStub.callCount).to.equal(3);
+        });
+      });
+
+      it('successfully loads 2 more facilities on click', async () => {
+        const { props, mockStore } = getData({});
+        const { selectors, getByText, container } = subject({
+          props,
+          mockStore,
+        });
+
+        mapboxStub.resolves(mapBoxSuccessResponse);
+        facilitiesStub
+          .onFirstCall()
+          .resolves(mockFetchChildFacilityWithCaregiverSupportResponse);
+        facilitiesStub.onSecondCall().resolves(mockFetchFacilitiesResponse);
+
+        await waitFor(() => {
+          inputVaSearchInput(container, 'Tampa', selectors().input);
+          expect(selectors().loader).to.exist;
+        });
+
+        await waitFor(() => {
```

Comment:
This may be worth doing in a general refactor for performance later across all specs, but I think we should look at doing a desctructure of the `selectors()` call. By repeating the call, we're repeating the DOM queries over and over again. Albeit small, we're certainly taking a performance hit by repeating those queries.

Reply:
Yeah, I agree with you. That's definitely a refactor I'd like to look into, although probably in it's own PR.